src/schema/controllers/user-controller.js

```diff
@@ -1,20 +1,19 @@
 import db from '../../models'
 
-export default {
-  createUser: function createUser (args) {
-    const data = args.input
-    return db.user.create({
-      firstName: data.firstName,
-      lastName: data.lastName,
-      username: data.username,
-      password: data.password
-    })
-  },
-  deleteUser: function deleteUser (args) {
-    return db.user.destroy({
-      where: {
-        id: args.id
-      }
-    })
-  }
+export function createUser (args) {
+  const data = args.input
```

Comment:
You're copying the whole content of args.input into a new variable. What's the value of it? if it's clarity, it would be better to destruct it and create an alias for it:
```javascript
export const createUser = ({ input: data }) => {
  //...
```

Reply:
Thanks, I wanted to detach args from input since I thought that args.input.field would be confusing and I didn't know how to do it in a clearer way.